Makefile.work

```diff
@@ -51,12 +51,14 @@ comma := ,
 
 ifeq ($(shell uname -m), aarch64)
 COMPILE_HOST_ARCH := arm64
-else ifeq ($(shell uname -m), armv7l)
+else ifneq (,$(filter $(shell uname -m),armv7l armv8l))
 COMPILE_HOST_ARCH := armhf
 else
 COMPILE_HOST_ARCH := amd64
 endif
 
+COMPILE_HOST_ARCHS := $(COMPILE_HOST_ARCH) $(HOST_ARCHS)
```

Comment:
>HOST_ARCHS [](http://example.com/codeflow?start=45&length=10)

not defined?

Reply:
I search the repo and could not found `HOST_ARCHS`

Reply:
Yes, it is not defined, normally, we are not necessary to define it, the default value is empty.
If we want to support build armhf on arm64, we can set HOST_ARCHS=armhf, it means that the architecture armhf is supported on arm64, although it does not match with "uname -m".

Reply:
OK, this paragraph is good code comment, could you add it? Otherwise user could not easily figure out this hidden feature.